ui/src/views/settings/SettingsView.tsx

```diff
@@ -225,6 +225,22 @@ export default class SettingsView extends React.Component<Props> {
                 onChange={value => this.updateSetting('openWindowOnStartup', value)}
               />
             </Form.Item>
+            <Form.Item label="Force quit on close">
+              <Switch
+                checked={settings.quitOnClose}
+                onChange={value => this.updateSetting('quitOnClose', value)}
+              />
+            </Form.Item>
+            <p>
+              To use scheduling feature, this settings <strong>must be</strong> disabled.
```

Comment:
nit: technically untrue as the app could still post as long as the window was minimized, although I understand the purpose. Probably fine to keep as is for now or just suggest that it is recommended to disable when using scheduled posts.

Reply:
Fixed!